version.py

```diff
@@ -31,17 +31,15 @@
 import logging
 import subprocess
 
-# Modify the following two settings during release
+# Modify the following value during release
 # ---------------------------------------------------
 # Current version
 # We use the version of the incoming release for code
-# that is under development
+# that is under development. It is also fallback version
```

Comment:
Maybe change "It is also..." to something like "Version below will be used as a fallback version when the version can't be determined via 'git describe"?

Reply:
Actually the use of `git describe` is not the default, so this statement doesn't apply here. This script covers also the use case of generating a release form a completely non-git source base, e.g. a source-only tarball.

Reply:
btw, I expanded this comment, trying to make it a bit more clear.

Reply:
@leandron ah, got it. ok. Thanks! The second version for this comment (and the other additional ones) is really helpful!